app/controllers/revision_feedback_controller.rb

```diff
@@ -16,13 +16,13 @@ def index
   private
 
   def set_latest_revision_id
-    @article = Article.find(params[:article_id])
-
-    query = { prop: 'revisions', pageids: @article.mw_page_id, rvprop: 'ids' }
+    query = { prop: 'revisions', titles: params['title'], rvprop: 'ids' }
     @wiki = Wiki.find_by(language: 'en', project: 'wikipedia')
     response = WikiApi.new(@wiki).query(query)
-    results = response&.data
-    revisions = results.dig('pages', @article.mw_page_id.to_s, 'revisions')
+    page = response&.data['pages']
```

Comment:
The safe navigation operator here doesn't help. If response is nil, doing `page.keys` below will still cause an error.